lib/core/test/unit/Cardano/Wallet/DB/Arbitrary.hs

```diff
@@ -23,6 +23,8 @@ module Cardano.Wallet.DB.Arbitrary
     , GenState
     , MockChain (..)
     , InitialCheckpoint (..)
+    , StakePoolsFixture (..)
+    , once
```

Comment:
Let's move that in a separate module `Cardano.Pool.DB.Arbitrary` and keep things separate :pray:

Reply:
moved